roles/hydra-stack/install/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
 #
 - include: tomcat.yml
 - include: solr_5.yml
-  when: solr_version | match("^5\.[0-9]\.[0-9]")
+  when: solr_version | match("^5\.\d{1,2}\.{1,2}")
```

Comment:
Mark's PR for camper made me think, how about changing this to match >=5 so it will install Solr 6 as well - I don't think the install process will change again for a while, so including anything greater than 5.x seems like a lower-maintenance approach.